Spring-2019/linked_lists/SOLUTIONS_2019Spring.md

```diff
@@ -39,21 +39,35 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Rotate List 
 
-Source: TODO :bug:
+Source: [CrazyForCode](http://www.crazyforcode.com/rotate-linked-list-k-nodes/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a singly linked list, rotate the list by `k` nodes.
```

Comment:
Is it rotating it clock-wise? counter clock-wise?

Reply:
fixed in 775a59533001aa2787e743085c5b3c7c1ae69c34